markdown/3.x/en/api/ff-loading-spinner.api.md

```diff
@@ -0,0 +1,14 @@
+## `ff-loading-spinner`
+___
+### Properties
+| Name | Description |
+| ---- | ----------- |
+| **is-active**&nbsp;(Boolean) (Default: "false") | If set, the element is shown and animated. Otherwise the element is hidden. |
+| **stroke-color**&nbsp;(String) (Default: "#4285F4") | Takes a CSS color value and applies it to the spinning stroke. |
+| **manual**&nbsp;(Boolean) (Default: "false") | If set, the element will not subscribe to any default events and subsequently has to be shown or hidden manually | 
+
+### Methods
+| Name | Description |
+| ---- | ----------- |
+| **show** | Shows the element. Calling this method is equivalent to setting the `is-active` attribute. |
+| **hide** | Hides the element. Calling this method is equivalent to removing the `is-active` attribute. |
```

Comment:
Most places list methods with `()`. This is useful to explicitly show which parameters are available.
I know it's inconsistent at the moment, so we should clean up other places, too, some time.
```suggestion
| **show()** | Shows the element. Calling this method is equivalent to setting the `is-active` attribute. |
| **hide()** | Hides the element. Calling this method is equivalent to removing the `is-active` attribute. |
```